warp-dba/build.gradle.kts

```diff
@@ -0,0 +1,79 @@
+
+plugins {
+    id("com.android.library")
+    id("org.jetbrains.kotlin.android")
+    id("maven-publish")
+}
+apply(plugin = "com.jfrog.artifactory")
+
+android {
+    namespace = ConfigData.namespaceDba
+    compileSdk = ConfigData.compileSdkVersion
+
+    defaultConfig {
+        minSdk = ConfigData.minSdkVersion
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        consumerProguardFiles("consumer-rules.pro")
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+        debug {
+            isMinifyEnabled = false
+        }
+    }
+    testOptions.unitTests {
+        isIncludeAndroidResources = true
+    }
+    compileOptions {
```

Comment:
Having a gradle script to add basic stuff like java version and compose could be useful to make it less verbose